contrib/python/tests/python/pants_test/contrib/python/checks/tasks/checkstyle/test_checker.py

```diff
@@ -12,8 +12,8 @@
 from pants_test.backend.python.tasks.python_task_test_base import PythonTaskTestBase
 
 from pants.contrib.python.checks.tasks.checkstyle.checker import PythonCheckStyleTask
-from pants.contrib.python.checks.tasks.checkstyle.print_statements_subsystem import \
-  PrintStatementsSubsystem
+from pants.contrib.python.checks.tasks.checkstyle.variable_names_subsystem import \
+  VariableNamesSubsystem
```

Comment:
The Print Statements tests don't work as expected on Py3. Because they raise a syntax error with Python 3, the tests weren't working as intended.

So, I changed to using Variable Names. The specific error doesn't matter much - this file tests the `checker` mechanics, and there are other files to check each specific subsystem.